std/file.d

```diff
@@ -620,27 +620,70 @@ private void renameImpl(const(char)[] f, const(char)[] t, const(FSChar)* fromz,
 
 /***************************************************
 Delete file $(D name).
+
+Params:
+    name = string or range of characters representing the file name
+
 Throws: $(D FileException) on error.
  */
-void remove(in char[] name) @trusted
+void remove(R)(R name)
+    if (isInputRange!R && isSomeChar!(ElementEncodingType!R) || isSomeString!R)
```

Comment:
Forgot to mention - this is enough: `if (isInputRange!R && isSomeChar!(ElementEncodingType!R))`